src/Header.php

```diff
@@ -0,0 +1,35 @@
+<?php namespace DeSmart\Mailer;
+
+class Header
+{
+    /** @var string */
+    protected $name;
+    /** @var string */
+    protected $value;
+
+    /**
+     * @param string $name
+     * @param string $value
+     */
+    public function __construct($name, $value)
+    {
+        $this->name = $name;
```

Comment:
Hm, I think I'll rewrite it, because SendGrid has totally different structure when it comes to elements like reply-to. I have think about it more.